src/OfflineQuery.js

```diff
@@ -538,7 +538,7 @@ function matchesKeyConstraints(className, object, objects, key, constraints) {
         const distance = point.radiansTo(centerPoint);
         return distance <= maxDistance;
       }
-      break;
+      return false;
```

Comment:
Why this change? Seems to go beyond "test coverage" - is this a bug fix?

Reply:
It’s technically unreachable code. The change is for the test to see what happens if it is reached

Reply:
Are you sure it's unreachable? I haven't looked much into this, but what if the constraint is neither `$polygon` nor `$centerSphere`?

Reply:
Using the SDK as is it’s unreachable. You could mess around with the internals to reach it like I do with the tests

Reply:
Got it, and have you looked into what the difference is between `break` and `return false`? For example `case '$maxDistance'` breaks, it doesn't return false, so maybe there is a reason why this here has a `break` and is not returning false?

Reply:
maxDistance is ignored so the break statement  falls to true (matching) in geoWithin this would be non matching invalid query and should return false